lib/tasks/schools_from_yml.rake

```diff
@@ -0,0 +1,30 @@
+namespace :schools do
+  desc "Reads from the ./config/code_schools.yml and creates records in the database."
+  task populate: :environment do 
+    schools =  YAML::load_file(File.join("./config", "code_schools.yml"), "r")    
+    schools.each do |school|
+      object = CodeSchool.new(
+        name: school["name"], 
+        url: school["url"], 
+        logo: school["logo"], 
+        full_time: school["full_time"],
+        hardware_included: school["hardware_included"],
+        has_online: school["has_online"],
+        online_only: school["online_only"],
+        notes: school["notes"]
+        )
+      object.save
```

Comment:
Perhaps consider using the `create!` method for creating the `CodeSchool` and `Location` records.  This will prevent any silent failures, and force the raising of an error, should there be an issue.  Then you can `rescue` and `log` those errors.  

Then you'll be able to identify any records that were not created, so you can go back and troubleshoot, manually add them, etc.

Here is [an example](https://github.com/OperationCode/operationcode_backend/blob/master/lib/tasks/resource.rake) of this.